packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -57,6 +57,9 @@ function validateNextProps(props, nextProps) {
  * @propType {string} appId - The Algolia application id.
  * @propType {string} apiKey - Your Algolia Search-Only API key.
  * @propType {string} indexName - The index in which to search.
+ * @propType {function=identity} configureSearchParameters - function to tweak the parameters sent to Algolia.
```

Comment:
Definitely we should explain what identity means because I'm sure some people will not know, and that can be frustrating. Your explanation is fine, and associated with an example (on the guide you mention @vvo ?), it would be imho nice :)